exporter/signalfxexporter/README.md

```diff
@@ -90,6 +90,24 @@ that are allowed to be used as a dimension key in addition to alphanumeric
 characters. Each nonalphanumeric dimension key character that isn't in this string 
 will be replaced with a `_`.
 - `max_connections` (default = 100):  The maximum number of idle HTTP connection the exporter can keep open.
+- `ingest_tls`: (no default) exposes a list of TLS settings that user can adjust.
```

Comment:
Please mention that it's needed only in case when secure connection needs to be established between agent and the gateway

Reply:
done.thx